### PR TITLE
Feature: add {jelos} output token

### PR DESCRIPTION
**Pull request by samwelnella**
Updates gameConsole.ts, argumentsParser.ts and outputFactory.ts to add a {jelos} output token to support JELOS' directory structure

### CODE REVIEW

File: src/types/outputFactory.ts [hunk 1]
```diff
@@ -269,6 +269,11 @@ export default class OutputFactory {
     if (batocera) {
       output = output.replace('{batocera}', batocera);
     }
+
+    const jelos = gameConsole.getJelos();
```
Comment (emmercm) [inline src/types/outputFactory.ts:273]: This is great work! I spot-checked a number of the values and they look right. One last step, could you add some unit tests for this in `outputFactory.test.ts`? There are some other examples in there that you can copy.

**Review by emmercm — CHANGES_REQUESTED** [review]

**Comment by samwelnella** [comment]
Hi, I've added new tests in `outputFactory.test.ts` for `{jelos}` based on the `{batocera}` tests

**Review by emmercm — APPROVED** [review]

**Comment by emmercm** [comment]
Hi @samwelnella, looks like there are some syntax errors reported by CI.

**Comment by samwelnella** [comment]
I think I fixed the issue but some tests have been stuck with "Expected" status for a while. I'm not sure what to do.

**Comment by emmercm** [comment]
Hi @samwelnella, it's entirely a GitHub security setting that doesn't have fine-grained control. Users that aren't marked as repository collaborators can't initiate some workflows, to prevent secrets from being exported by malicious actors. I'll make sure the tests get triggered.

**Comment by emmercm** [comment]
@samwelnella I'm fighting some unit tests that keep failing in macOS, I'm going to need to debug. I think there's a potential bug being caused by https://github.com/emmercm/igir/pull/630.

**Pull request merged**